core/src/trezor/utils.py

```diff
@@ -16,6 +16,7 @@
     USE_BUTTON,
     USE_HAPTIC,
     USE_OPTIGA,
+    USE_POWER_MANAGER,
```

Comment:
Consider also adding to `core/site_scons/site_tools/micropython/__init__.py` for better dead code elimination.